pkg/common/cns-lib/volume/listview.go

```diff
@@ -135,15 +136,38 @@ func getListViewWaitFilter(listView *view.ListView) *property.WaitFilter {
 	return filter
 }
 
+func (l *ListViewImpl) isSessionValid(ctx context.Context) bool {
+	l.mu.Lock()
+	defer l.mu.Unlock()
+	log := logger.GetLogger(ctx)
+	// If session hasn't expired, nothing to do.
+	sessionMgr := session.NewManager(l.virtualCenter.Client.Client)
```

Comment:
add a non nil check for `l.virtualCenter.Client`

Reply:
Added.